ckanext/cfpb_extrafields/validators.py

```diff
@@ -1,18 +1,59 @@
 import re
 
-PRA_CONTROL_NUM_REGEX = re.compile('^\d{4}-\d{4}$')
-
 # Abstracted to simplify unit testing via mock
 def Invalid(message):
     import ckan.plugins.toolkit as tk
     raise tk.Invalid(message)
 
+def dedupe_unordered(items):
+    return list(set(items))
+def contains_bad_chars(str):
+    ''' contains \"{},'''
+    test = False
+    for c in ['>','<']:
+        test = test or (c in str)
```

Comment:
haha, maybe a little too clever, I should probably just do if c in str: return True